seacas: 2025-08-19 release

gsjaardema opened the pull request:
See https://github.com/sandialabs/seacas/releases/tag/v2025-08-19 for details

<!--  
Remember that `spackbot` can help with your PR in multiple ways:
- `@spackbot help` shows all the commands that are currently available
- `@spackbot fix style` tries to push a commit to fix style issues in this PR
- `@spackbot re-run pipeline` runs the pipelines again, if you have write access to the repository 
-->

gsjaardema committed:
seacas: 2025-08-19 release

See https://github.com/sandialabs/seacas/releases/tag/v2025-08-19 for details

wdconinc reviewed (dismissed):
Based on upstream diff, indeed no changes that need to be applied here. Thanks!

gsjaardema commented:
Changes needed to 2025-08-19 so new release today.  Skip this version...

gsjaardema committed:
Update checksum

Discovered a couple files that needed to be changed in the release.  Updated files, tag, release and checksum.

gsjaardema commented:
Updated checksum for current release.  This should be good now.

gsjaardema commented:
@spackbot help

gsjaardema commented:
@spackbot fix style

gsjaardema committed:
Merge branch 'develop' into patch-3

gsjaardema committed:
[@spackbot] updating style on behalf of gsjaardema

gsjaardema commented:
@spackbot re-run pipeline
